Make file execution failing for windows.

### Environment
<!-- Please include if you've confirmed one version of something works while another one does not -->
- `drf-user` version(s) (`drf-user --version`:
- Operating System(s): Windows
- Python version(s): 3.8

### Description of the bug

Make install is working but make test and livehtml is not working.

### What you expected to happen
<!-- A clear and concise description of what you expected to happen. -->

Tests passing and html page constructed.

### How to reproduce (as minimally and precisely as possible)
<!-- If applicable, add screenshots to help explain your problem. -->
After cloning repo and install make using choco and run the make files getting this error

```
make test
${VIRTUAL_ENV-.venv}/bin/python -m pytest --disable-pytest-warnings --ds=tests.settings --cov=drf_user tests/
process_begin: CreateProcess(NULL, ${VIRTUAL_ENV-.venv}/bin/python -m pytest --disable-pytest-warnings --ds=tests.settings --cov=drf_user tests/, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:67: test] Error 2
```
### Anything else we need to know?
<!-- Add any other context about the problem here. -->

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -152,16 +152,6 @@ files for some of their features (e.g. automatically picking up the
 
 ## Local Development Environment
 
-Steps to Install make for windows
-
-- Install chocolatey from [here](https://chocolatey.org/install).
-
-- And then run following command
-
-```sh
-$ choco install make
-```
-
 <!-- You can (and should) run our test suite using [tox](https://tox.readthedocs.io/). However, you’ll probably want a more traditional environment as well. We highly recommend to develop using the latest Python 3 release because `interrogate` tries to take advantage of modern features whenever possible. -->
 
 First create a [virtual environment](https://virtualenv.pypa.io/). It’s out of
@@ -189,6 +179,12 @@ Change into the newly created directory and **after activating your virtual
 environment** install an editable version of `drf-user` along with its tests,
 docs requirements and to avoid committing code that violates our style guide, we
 use [pre-commit](https://pre-commit.com/) hooks:
+
+Note : For **windows** users, please refer this link to install choco [here](https://chocolatey.org/install) and then use following command to setup make
+
+```sh
+$ choco install make
+```
 
 ```sh
 (env) $ cd drf-user

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -152,6 +152,16 @@ files for some of their features (e.g. automatically picking up the
 
 ## Local Development Environment
 
+Steps to Install make for windows
+
+- Install chocolatey from [here](https://chocolatey.org/install).
+
+- And then run following command
+
+```sh
+$ choco install make
+```
+
 <!-- You can (and should) run our test suite using [tox](https://tox.readthedocs.io/). However, you’ll probably want a more traditional environment as well. We highly recommend to develop using the latest Python 3 release because `interrogate` tries to take advantage of modern features whenever possible. -->
 
 First create a [virtual environment](https://virtualenv.pypa.io/). It’s out of

--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,12 @@
 .PHONY: clean-pyc clean-build docs clean
 
-VENV = venv
-PYTHON=$(VENV)/bin/python
+VIRTUALENV = virtualenv --python=python3
+PYTHON = $(VENV)/bin/python
+VENV := $(shell echo $${VIRTUAL_ENV-.venv})
 
 # make it work on windows too
 ifeq ($(OS), Windows_NT)
+	VENV = venv
     PYTHON=$(VENV)/Scripts/python
 endif
 

--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,12 @@
 .PHONY: clean-pyc clean-build docs clean
 
-VIRTUALENV = virtualenv --python=python3
-PYTHON = $(VENV)/bin/python
-VENV := $(shell echo $${VIRTUAL_ENV-.venv})
+VENV = venv
+PYTHON=$(VENV)/bin/python
+
+# make it work on windows too
+ifeq ($(OS), Windows_NT)
+    PYTHON=$(VENV)/Scripts/python
+endif
 
 TEST_FLAGS=--verbosity=2
 COVER_FLAGS=--source=drf_user

